src/repart/repart.c

```diff
@@ -2675,6 +2675,17 @@ static int determine_current_padding(
         return 0;
 }
 
+static int verify_regular_or_block(int fd) {
+        struct stat st;
+
+        assert(fd >= 0 || fd == AT_FDCWD);
```

Comment:
this cannot really be AT_FDCWD. And if it was, then plain old fstat() wouldn't work on it anyway. please drop the right hand side.